### DISCUSSION TITLE
Any plan on migrating to GA4

### DISCUSSION THREAD

**Discussion by owenxu10** [ppoffice/hexo-theme-icarus#1078] [Q&A]
Google Analytics just annonuced last month that GA4 will replace the current Universal Analytics. (https://support.google.com/analytics/answer/11583528?hl=en)

On July 1, 2023, standard Universal Analytics properties will stop processing new hits. If you still rely on Universal Analytics, we recommend that you [prepare to use Google Analytics 4](https://support.google.com/analytics/answer/10759417) going forward.

Is there any plan to migrate to GA4?

**Comment by ppoffice** [comment]
Please create an issue in https://github.com/ppoffice/hexo-component-inferno if you rely on this plugin.
